pep-9999.rst

```diff
@@ -0,0 +1,1228 @@
+above, there are two related problems that risk breaking an
+externally-managed Python: you can install an incompatible
+new version of a package system-wide (e.g., with ``sudo pip
+install``), and you can install one in your user account
+alone, but in a location that is on the standard Python
+command's ``sys.path``
+(e.g., with ``pip install --user``). If the marker file were
+in the system-wide ``site-packages`` directory, it would not
+clearly apply to the second case. The `Alternatives`_
+section has further discussion of possible locations.
+
+The second behavior change takes advantage of the existing
+``sysconfig`` setup in distributions that have already
+encountered this class of problem, and specifically
+addresses the problem of a Python-specific package manager
+deleting or overwriting files that are owned by an external
+package manager.
+
+Use cases
+---------
+
+The changed behavior in this PEP is intended to "do the
+right thing" for as many use cases as possible.  In this
+section, we consider the changes specified by this PEP for
+several representative use cases / contexts. Specifically, we
+ask about the two behaviors that could be changed by this
+PEP:
+
+1. Will a Python-specific installer tool like ``pip
+   install`` permit installations by default, after
+   implementation of this PEP?
+
+2. If you do run such a tool, should it be willing to delete
+   packages shipped by the external (non-Python-specific)
+   package manager for that context, such as a distribution
+   package manager?
+
+(For simplicity, this section discusses pip as the
+Python-specific installer tool, though the analysis should
+apply equally to any other Python-specific package
+management tool.)
+
+This table summarizes the use cases discussed in detail
+below:
+
+==== ================================= =========================== ===================================================
+Case Description                       ``pip install`` permitted   Deleting externally-installed packages permitted
+==== ================================= =========================== ===================================================
+1    Unpatched CPython                 Currently yes; stays yes    Currently yes; stays yes
+2    Distribution ``/usr/bin/python3`` Currently yes; becomes no   Currently yes (except on Debian); becomes no
+                                       (assuming the distribution
+                                       adds a marker file)
+3    Distribution Python in venv       Currently yes; stays yes    There are no externally-installed packages
+4    Distribution Python in venv       Currently yes; stays yes    Currently no; stays no
+     with ``--system-site-packages``
+5    Distribution Python in Docker     Currently yes; stays yes    Currently yes; becomes no
+                                       (assuming the Docker image
+                                       removes the marker file)
```

Comment:
I think I want to lean towards expecting distributions / Docker base image providers to remove this marker file, because people are gonna get mad about not being able to `pip install` in a Docker container without a virtualenv (people already don't like the warning they now get :) ).